src/server/global_notifier.cpp

```diff
@@ -166,8 +164,8 @@ void GlobalNotifier::Impl::register_realm(sync::ObjectID id, StringData path) {
             auto config = info->coordinator->get_config();
             config.force_sync_history = true; // FIXME: needed?
             config.schema = util::none;
-            Realm::open_with_config(config, info->history, info->shared_group, read_only_group, nullptr);
-            info->shared_group->begin_read(old_version);
+            auto realm = info->coordinator->get_realm();
```

Comment:
We don't want to create a `Realm` here. It adds overhead and makes all of the other code more complicated because it's written to work with the Transaction directly.